Modules/IndividualAssessment/classes/class.ilIndividualAssessmentDataSet.php

```diff
@@ -12,14 +12,18 @@ public function getSupportedVersions() : array
         return ['5.2.0', '5.3.0'];
     }
 
+    // PHP8-Review: Method visibility should not be overridden
+    // PHP8-Review: Parameter's name changed during inheritance
```

Comment:
This is also interesting. Parameter names didn't matter a long time in PHP, as long as the signature was intact. [Now they might start to matter with PHP 8](https://wiki.php.net/rfc/named_params). I will as the @ILIAS-eLearning/technical-board for some advice if and how we would want to use named params in ILIAS.